pyproject.toml

```diff
@@ -33,35 +33,29 @@ title_format = "v{version}"
 underlines = "-~"
 issue_format = ":issue:`{issue}`"
 
-    [[tool.towncrier.type]]
-    directory = "breaking"
-    name = "Breaking Changes"
-    showcontent = true
-
-    [[tool.towncrier.type]]
-    directory = "deprecate"
-    name = "Deprecations"
-    showcontent = true
-
-    [[tool.towncrier.type]]
-    directory = "feature"
-    name = "New Features"
-    showcontent = true
-
-    [[tool.towncrier.type]]
-    directory = "bugfix"
-    name = "Bug Fixes"
-    showcontent = true
-
-    [[tool.towncrier.type]]
-    directory = "doc"
-    name = "Documentation"
-    showcontent = true
-
-    [[tool.towncrier.type]]
-    directory = "misc"
-    name = "Miscellaneous"
-    showcontent = true
+[tool.towncrier.fragment.breaking]
```

Comment:
We should keep the old array config here, since toml mappings apparently aren't ordered (and towncrier sorts them alphabetically).
see https://github.com/twisted/towncrier/issues/437